react/features/toolbox/components/web/Toolbox.tsx

```diff
@@ -348,9 +348,10 @@ const styles = () => {
         contextMenu: {
             position: 'relative' as const,
             right: 'auto',
-            maxHeight: 'inherit',
             margin: 0,
-            marginBottom: '8px'
+            marginBottom: '8px',
+            maxHeight: "60vh",
```

Comment:
Too small. This would cause the menu to scroll even when there's plenty of space. I think `80vh` should do